src/offsetPath.js

```diff
@@ -18,12 +18,33 @@
       return undefined;
     } else if (ray !== null) {
       inputType = 'ray';
-      var rayInput = ray[1];
-      var rayInputDegrees = parseAngleAsDegrees(rayInput);
-      if (rayInputDegrees === null) {
-        return undefined;
+      var rayInput = ray[1].split(/\s+/);
+      var result = {type: inputType,
+                    input: 'none',
```

Comment:
Note that Divy is concurrently working on creating a translate for ray(angle). If we change the name in this patch, we'll have a merge conflict.